### DISCUSSION TITLE
Unable to get command module to display results

### DISCUSSION THREAD

**Discussion by k5map** [fastfetch-cli/fastfetch#1080] [Q&A] [answered]
I generated the config JSON file via "fastfetch --gen-config".  I then added the following in the modules section after the seperator:
    {
      "type": "command",
      "shell": "/bin/sh",
      "text": "cat /sys/firmware/devicetree/base/serial-number",
      "key": "S-N"
    },

When I run fastfetch nothing is displayed.  Is there a problem with my command section?

**Comment by CarterLi** [comment]
please try

```jsonc
{
"type": "host",
"format": "{serial}",
"key": "S-N"
},
```

**Comment by k5map** [comment]
@CarterLi ... When I enter what you provided, I get a blank response on the line after Host.

**Reply by CarterLi** [reply]
Does `cat /sys/firmware/devicetree/base/serial-number` without sudo work?

**Comment by k5map** [comment]
Yes... I can type this as a user on a RPi and it returns a string.  But I believe the string is not null terminated because the command prompt is shown next to the output.

**Comment by k5map** [comment, marked as answer]
@CarterLi ...  I owe you a BIG apology.  

I wasn't paying attention to where I was running fastfetch because I installed it on one of my Pi virtual machine.  So, no wonder it will not output a serial number :)
